src/main/scala/nl/knaw/dans/api/sword2/DepositHandler.scala

```diff
@@ -453,4 +453,31 @@ object DepositHandler {
 
     getFile(bagStoreSettings.baseDir, topDir, uuidDir, version)
   }
+
+  def writeToProperties(bagDir: File)(implicit settings: Settings, id: String): Try[Unit] = Try {
+    for {
+      bag <- getBag(bagDir)
+      resources <- getResources(bag)
```

Comment:
This was not the intention. `easy-ingest-flow` should set the resources when they are actually created in archival storage, not `easy-sword2`. At this point we cannot be sure that the archival copy has actually been created.